### ISSUE TITLE
Update the website

### ISSUE THREAD

**Issue by prajjwaldimri** [prajjwaldimri/Friend-App#38]

**Comment by RohanRon** [comment]
I will develop a new website with more information. This is purely me volunteering time for this amazing app and fully support it.

**Comment by RohanRon** [comment]
For the website I am planning on a page that lists all the people that have contributed to this. Would there be a list already available that I can copy-paste?

**Comment by prajjwaldimri** [comment]
Yes, the name of all current contributors are already in the app and I will also add them in the readme of this repo

**Comment by RohanRon** [comment]
@prajjwaldimri ok I have found it and will add to website soon.

**Comment by prajjwaldimri** [comment]
@RohanRon Here is the store badge for download links: 
`<a href="https://www.microsoft.com/store/apps/9nblggh4q864?ocid=badge"><img src="https://assets.windowsphone.com/f2f77ec7-9ba9-4850-9ebe-77e366d08adc/English_Get_it_Win_10_InvariantCulture_Default.png" alt="Get it on Windows 10" /></a>`

**Comment by RohanRon** [comment]
I've already started work on the website and will continue to update it in the coming weeks.
http://befriendapp.weebly.com/

I will see if I can get a free/low-cost domain because this app and all of us are not paid.

**Comment by prajjwaldimri** [comment]
You can host it on Github! Just update the gh-pages branch

**Comment by RohanRon** [comment]
sure :)

**Comment by RohanRon** [comment]
As you may have noticed I had recreated some sample images of the app. @prajjwaldimri here is the link to all the files https://1drv.ms/f/s!AnpZOvbrL3D6j-ttXdFPRaO7E8TVOQ

**Comment by RohanRon** [comment]
I had some difficulties with hosting website on GitHub so I've left it on Weebly (it's free and no cost). Though if we ever plan to get a domain like BeFriendapp.com with email like admin@BeFriendapp.com then these will cost us.

**Comment by prajjwaldimri** [comment]
@RohanRon Seems cool, thanks!
